### ISSUE TITLE
Contribute to unminified JS

### ISSUE THREAD

**Issue by ericbolo** [ludrao/django-tellme#40]
The screenshots made with django-tellme do not capture SVGs. I'd like to try and fix that but the source only contains minified JS which is hard to read. 

Is it possible to make the un-minified JS available so we can easily contribute ?

**Comment by ludrao** [comment]
Yes absolutely. The javascript is coming from upstream projects https://github.com/ivoviz/feedback and https://github.com/niklasvh/html2canvas. You could clone those repo do the fix and then rebuild (minify) to embedded it in here.
